src/components/search/index.js

```diff
@@ -78,6 +75,7 @@ export default function Search({ collapse }) {
   const ref = createRef()
   const [searchQuery, setQuery] = useState(``)
   const [focus, setFocus] = useState(false)
+  const { isDesktop, isMobile } = useBreakpoint()
```

Comment:
are these used now?

Reply:
not used anymore, i forgot to removed it. My bad

Reply:
Noworries, we run eslint in a GH action that is not reported in the PR... trying to fix that https://github.com/climatescape/climatescape.org/issues/152